src/applications/check-in/day-of/pages/TravelReview.jsx

```diff
@@ -118,8 +114,8 @@ const TravelQuestion = props => {
       <div className="vads-u-margin-top--2p5">
         <a
           className="vads-u-margin-left--auto"
-          href="travel-address"
-          onClick={e => onEditClick(e)}
+          href="travel-mileage"
+          onClick={() => jumpToPage('/travel-mileage')}
```

Comment:
Sorry I led you astray here. The Figma says to go to `travel-mileage`, but that will be what we want after re-ordering. For right now, this should probably go back to `travel-question`.

The other thing you will want to do here is add a `e.preventDefault();` right now it is triggering a navigation which reloads the app and re-fetches everything.